lib/matplotlib/projections/polar.py

```diff
@@ -112,10 +112,6 @@ def get_matrix(self):
         return self._mtx
     get_matrix.__doc__ = Affine2DBase.get_matrix.__doc__
 
-    def __getstate__(self):
-        return {}
-
-
```

Comment:
Nit-pick: We leave 2 newlines between top level module objects. (I think the PEP8 tests will probably catch this)